po/es.po

```diff
@@ -1597,40 +1590,40 @@ msgstr "atributo no encontrado"
 
 #: src/tagsouppullparser.cpp:145
 msgid "EOF found while reading XML tag"
-msgstr "EOF encontrado al leer la etiqueta XML"
+msgstr "Fin de archivo (EOF) encontrado al leer una etiqueta XML"
 
 #: src/urlviewformaction.cpp:48 src/urlviewformaction.cpp:61
 msgid "No link selected!"
-msgstr "¡Ningún enlace seleccionado!"
+msgstr "Ningún enlace seleccionado."
 
 #: src/urlviewformaction.cpp:140
 msgid "Save Bookmark"
 msgstr "Guardar marcador"
 
 #: src/urlviewformaction.cpp:162
 msgid "URLs"
-msgstr "URLs"
+msgstr "URL"
```

Comment:
"Les URL", perhaps? (See above about similar string in ConfigContainer)

Reply:
Withdrawn since titles don't need articles, see https://github.com/newsboat/newsboat/pull/874#issuecomment-612441172.